### PR TITLE
Some files for 1.0.0.216 #283

### PR DESCRIPTION
**Pull request by FightofLife**
Hey, i´m new to civ modding/lua scriptting.
But i thought 'let´s try' - so i did some files.
Maybe it helps :)
```
Panels/citypanel.xml
Panels/citypaneloverview.xml
Popups/boostunlockedpopup.lua
Popups/mappinpopup.xml
Popups/techciviccompletedpopup.lua
Popups/wonderbuiltpopup.lua
Screens/techtree.lua
ToolTips/plottooltip.lua
productionpanel.xml
toppanel.lua
toppanel.xml
worldtracker.lua
worldtracker.xml
```
for #283

### CODE REVIEW

**Comment by Azurency** [comment]
Thank you for this contribution 👍, some duplicate with #285. 
Will have to review the changes and then you'll maybe need to remove some file if there're already merged, I'll tell you.

**Comment by Azurency** [comment]
I just gave a very quick look at your changes, and you sometime used tabs instead of 2 spaces could you try to fix this ?

**Comment by FightofLife** [comment]
Hey,
sry - yeah, ich will fix it!
(u want a "fixing-commit" or should i change the history of the branch? (guess if i change history i should create new PR - or?)

**Comment by Azurency** [comment]
Whatever you're more confortable with. You can change the history of your branch, it will automatically update the pull request I think. And a fix commit is not a problem, I'll squash the commits upon merge.

**Comment by FightofLife** [comment]
So i made a new commit where i changed all tabs by double space.
(if it was to much replacing tell me - then i will reset and just replace my tabs by spaces ;-) )

File: Assets/UI/Popups/techciviccompletedpopup.lua [hunk 1]
```diff
@@ -1,423 +1,436 @@
--- ===========================================================================
```
Comment (Azurency) [inline Assets/UI/Popups/techciviccompletedpopup.lua]: The changes on that file are too much for just 26 insertions and 13 deletions in the update. Can you, for this file here, just replace the modified parts and not change tabs to spaces in the whole file 😉

File: Assets/UI/ToolTips/plottooltip.lua [hunk 2]
```diff
@@ -531,7 +546,7 @@ function View(data:table, bIsUpdate:boolean)
       if (data.WonderType == nil) then
         table.insert(details, Locale.Lookup("LOC_TOOLTIP_PLOT_BUILDINGS_TEXT"));
       end
-			local greatWorksSection: table = {};
+      local greatWorksSection: table = {};
       for i, v in ipairs(data.BuildingNames) do
           if (data.WonderType == nil) then
```
Comment (Azurency) [inline Assets/UI/ToolTips/plottooltip.lua]: can you reindent this line (minus 2 spaces)

File: Assets/UI/ToolTips/plottooltip.lua [hunk 3]
```diff
@@ -545,14 +560,14 @@ function View(data:table, bIsUpdate:boolean)
...
-					end
-				end
+            table.insert(greatWorksSection, "- " .. Locale.Lookup(GameInfo.GreatWorks[greatWorkType].Name));
           end
-			if #greatWorksSection > 0 then
-				table.insert(details, Locale.Lookup("LOC_GREAT_WORKS") .. ":");
-				for i, v in ipairs(greatWorksSection) do
-					table.insert(details, v);
+        end
+          end
```
Comment (Azurency) [inline Assets/UI/ToolTips/plottooltip.lua]: and this one (minus 4 spaces).

File: Assets/UI/ToolTips/plottooltip.lua (hunk 2 above)
Comment (FightofLife) [reply Assets/UI/ToolTips/plottooltip.lua]: Hey,
you want to have the line "local greatWorksSection" (new line 549) to have two spaces less?
But why? It´s on the same indentation as the lines before and after.
Or am I understanding something wrong? :)

Comment (FightofLife) [reply Assets/UI/ToolTips/plottooltip.lua]: Ah ok - all fine. forget it, noticed u mean line 551 :)

**Comment by FightofLife** [comment]
Ok, i did the changes.

**Comment by Azurency** [comment]
Thank you for your help and contribution on the update 🎉 !

**Comment by FightofLife** [comment]
Hey Azurency, thank you for all your work :)

Next time i will be more focused on indentations!

**Pull request merged**